docs/console-api/models/layer_parameter.json

```diff
@@ -0,0 +1,38 @@
+{
+  "type": "object",
+  "x-examples": {
+    "example-1": {
+      "name": "a_parameter",
+      "type": "number",
+      "defaultValue": 100
+    }
+  },
+  "description": "Parameter object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "Name of this parameter."
+    },
+    "type": {
+      "type": "string",
+      "enum": [
+        "string",
+        "boolean",
+        "number",
+        "object",
+        "array"
+      ],
```

Comment:
super nit: I'd keep these in the same order, since they go together

Reply:
[61e8263](https://github.com/statsig-io/docs/pull/346/commits/61e8263e27c6414e7f2aaee7416bdfc532b446e3)